spec/controler_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'controler'
+require 'user_interface'
+
+RSpec.describe Controler do
+  let(:actions) { [double('NullAction')] }
+  let(:output) { StringIO.new }
+  let(:valid_input) { StringIO.new('1') }
+
+  it 'tells the user_interface to print the menu' do
+    ui = UserInterface.new(valid_input, output)
+    ctrl = Controler.new(ui, actions)
```

Comment:
what is `ctrl` 
control?
cat roll?
controller? 
:slightly_smiling_face: 

always prefer full words, noone will charge us money for extra letters :+1: